### ISSUE TITLE
Incorrect relative size using floating point numbers on style properties.

### ISSUE THREAD

**Issue by hell-oh-world** [renpy/renpy#2491]
I noticed that whenever I try to set the size of a displayable relatively like `xysize (0.5, 0.5)`, the calculations are off.
![image](https://user-images.githubusercontent.com/59941217/101445515-9ab10900-395c-11eb-9ea9-31ea74cb9835.png)
The code I used:
```py
frame:
        background "#f00"
        xysize (500, 100)
        align (0.5, 0.25)

        frame:
            background "#0f0"
            xysize (0.5, 0.5)
```
I realized that in the `render` method of the `Window` class, the xmin/xmax/ymin/ymax are all being scaled by the `scale` function unnecessarily.

**Comment by Gouvernathor** [comment]
Can't reproduce, seems fixed.

**Comment by renpytom** [comment]
There was a problem like this that was fixed sometime in the 7.4.x series. @hell-oh-world, which version of Ren'Py are you using?
